docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -1,2 +1,327 @@
+
+  In other general datasets, otx ends training at the right time without adjusting ``num_iters``.
+
+.. note::
+
+  The training results are ``weights.pth`` and ``label_schema.json`` files that located in ``otx-workspace-INSTANCE_SEGMENTATION/models`` folder, while training logs and tf_logs for `Tensorboard` visualization can be found in the ``otx-workspace-INSTANCE_SEGMENTATION`` dir.
+
+  ``weights.pth`` and ``label_schema.json``, which are needed as input for the further commands: ``export``, ``eval``,  ``optimize``,  etc.
+
+.. code-block::
+
+  ...
+  2023-02-21 22:34:53,474 | INFO : Update LrUpdaterHook patience: 5 -> 2
+  2023-02-21 22:34:53,474 | INFO : Update CheckpointHook interval: 1 -> 5
+  2023-02-21 22:34:53,474 | INFO : Update EvalHook interval: 1 -> 5
+  2023-02-21 22:34:53,474 | INFO : Update EarlyStoppingHook patience: 10 -> 3
+  2023-02-21 22:34:54,320 | INFO : Epoch [1][2/2] lr: 3.400e-04, eta: 3:14:44, time: 1.180, data_time: 0.784, memory: 7322, current_iters: 1, loss_rpn_cls: 0.0720, loss_rpn_bbox: 0.0250, loss_cls: 2.6643, acc: 89.3066, loss_bbox: 0.3984, loss_mask: 3.5540, loss: 6.7136, grad_norm: 66.2921
+
+  ...
+  2023-02-21 22:35:07,908 | INFO : Inference completed
+  2023-02-21 22:35:07,908 | INFO : called evaluate()
+  2023-02-21 22:35:07,909 | INFO : F-measure after evaluation: 0.33333333333333326
+  2023-02-21 22:35:07,909 | INFO : Evaluation completed
+  Performance(score: 0.33333333333333326, dashboard: (1 metric groups))
+
+After that we have the PyTorch instance segmentation model trained with OTX, that we can use for evaliation, export, optimization and deployment.
+
+***********
+Validation
+***********
+
+1. ``otx eval`` runs evaluation of a trained
+model on a specific dataset.
+
+The eval function receives test annotation information and model snapshot, trained in the previous step.
+Please note, ``label_schema.json`` file contains meta-information about the dataset and it should be located in the same folder as the model snapshot.
+
+``otx eval`` will output a F-measure for instance segmentation.
+
+2. The command below will run validation on our dataset
+and save performance results in ``performance.json`` file:
+
+.. code-block::
+
+  (otx) ...$ otx eval --test-data-roots /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      --load-weights models/weights.pth \
+                      --save-performance performance.json
+
+We will get a similar to this validation output:
+
+.. code-block::
+
+  ...
+
+  2023-02-21 22:37:10,263 | INFO : Inference completed
+  2023-02-21 22:37:10,263 | INFO : called evaluate()
+  2023-02-21 22:37:10,265 | INFO : F-measure after evaluation: 0.33333333333333326
+  2023-02-21 22:37:10,265 | INFO : Evaluation completed
+  Performance(score: 0.33333333333333326, dashboard: (1 metric groups))
+
+.. note::
+
+  You can omit ``--test-data-roots`` if you are currently inside a workspace and have test-data stuff written in ``data.yaml``.
```

Comment:
Is it working? Because I remember that `otx eval` required for me "--test-data-roots" and "--load-weights" paths even inside workspace

Reply:
Yes this works fine.